next.config.js

```diff
@@ -190,7 +190,7 @@ const sentryWebpackPluginOptions = {
 
   org: "mozilla",
   project: "firefox-monitor",
-  silent: true, // Suppresses all logs
+  silent: false, // Suppresses all logs
```

Comment:
Also, why this change? Was this related to sourcemaps, or was for debugging?

Reply:
I think it makes sense to leave this on, it only runs when tags are applied (or if `UPLOAD_SENTRY_SOURCEMAPS` is true for some other reason). It's a lot more work to verify if it's working otherwise.